components/subHeader/subHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from "./subHeader.module.css";
+
+export type SubHeaderProps = {
+  infoOne: string;
+  infoOneNumber: string;
+  infoTwo: string;
+  infoTwoNumber: string;
+  infoThree: string;
+  infoThreeNumber: string;
```

Comment:
If you convert the amounts into arrays, the use becomes more variable: You can use the numbers for calculations and maybe even use different currencies at some point. You have more control, too :)
```suggestion
  infoTwoNumber: [number, string];
  infoThree: string;
  infoThreeNumber: [number, string];
```
I'm no pro in TS, but I think you can do it like this :)